website/docs/framework-docs/intro.mdx

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 
 ## What is Boyka Framework?
 
-Boyka Framework is an open source Test Automation Framework based in Java, which helps you in automating Web, API, Android and iOS based applications with ease.
+Boyka Framework is an open source Test Automation Framework written in Java, that helps you in automating Web, API, Android and iOS based applications with ease.
```

Comment:
Boyka Framework is an open source Test Automation Framework written in Java that supports automating Web, API, and Mobile(Android and iOS applications)  seamlessly.